assets/blocks/course-list-filter-block/course-list-filter-edit.js

```diff
@@ -109,6 +109,9 @@ function CourseListFilter( {
 	const blockProps = useBlockProps();
 
 	if ( 'course' !== query?.postType ) {
+		setAttributes( {
```

Comment:
I believe @LavaGolem solved this problem already for course categories, only in a different way. Would that solution work here? I like that it doesn't introduce an extra CSS class.

Reply:
Yeah, actually I suggested her to fix it that way on design channel :p I tried the same thing here too, but we have a flex class on this block's default parent class, which was actually causing the issue here. Category block's default alignment was left, so setting the align attribute to false/center was fixing it there. In our case the alignment is okay, but the flex tries to align the warning box the same way it aligns the dropdowns. Adding the conditional class felt like the simplest solution till now, but I can keep looking for a simpler/better one ☺️ Should I do that or keep it as it is now?

Thanks!

Reply:
OK, the only thing is that it doesn't seem to be working for me on either Divi or Astra. They're still misaligned in the editor. 🤔

Reply:
Thanks for rechecking @donnapep! Updated a bit [here](https://github.com/Automattic/sensei/pull/5617/commits/8c8c771d0187b6ffdc0a25b50ad299b6b30f6466), can you kindly check if it aligns properly now? I've checked locally on Astra and Divi. Hopefully will work for you too ☺️